docs/changelog/1_0_a3.rst

```diff
@@ -0,0 +1,137 @@
+===========
+1.0 Alpha 3
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 3.
+
+
+CLI
+===
+
+We've converted EdgeDB command-line tools to using Rust. The tools are
+a single binary now and can run on multiple different platforms,
+including Windows.
+
+Notably the REPL has been converted and now features more commands:
+
+.. code-block::
+
+      \dump FILENAME           dump current database into a file
+      \restore FILENAME        restore the database from file into
+                               the current one
+      \create-database DBNAME  create a new database
+      \s, \history             show history
+      \e, \edit [N]            spawn $EDITOR to edit history entry N
+                               then use the output as the input
+      \set [OPTION [VALUE]]    show/change setting
+      \set                     Show setting descriptions (without
+                               arguments)
+
+Existing introspection commands have been slightly updated to use a
+different way of supplying command options. Commands now also have
+long-form names that are more human-readable.
+
+Also, there are a few settings that control the look and feel of the
+REPL (and can be changed via the ``\set`` command:
+
+.. code-block::
+
+    expand-strings             Stop escaping newlines in quoted
+                               strings
+    implicit-properties        Print implicit properties of objects:
+                               id, type id
+    input-mode                 Set input mode. One of: vi, emacs
+    introspect-types           Print type names instead of `Object`
+                               in default output mode (may fail if
+                               schema is updated after enabling
+                               option)
+    limit                      Set implicit LIMIT. Defaults to 100,
+                               specify 0 to disable
+    output-mode                Set output mode. One of: json, json-
+                               elements, default, tab-separated
+    verbose-errors             Print all errors with maximum
+                               verbosity
+
+
+EdgeQL
+======
+
+* Add ``+=`` and ``-=`` operations for use in :eql:stmt:`UPDATE`
+  (:eql:gh:`#165`).
+* Distinguish ``REQUIRED`` and ``OPTIONAL`` query parameters
+  (:eql:gh:`#1352`, :eql:gh:`#1355`)
+* Allow tuple arrays in schema definitions (:eql:gh:`73125882`).
+* Get rid of ``schema::CompoundType`` in favor of ``is_compound_type``
+  property (:eql:gh:`c30c1c4f`).
+* Add :eql:type:`anytype` and :eql:type:`anytuple` as proper abstract
+  types to the schema (:eql:gh:`affb65de`).
+* ``schema::Constraint`` no longer has ``args``, use ``params`` instead
+  (:eql:gh:`14cec6f1`).
+* Make ``std::Object`` the implicit base type for all user-defined
+  object types only.
```

Comment:
```suggestion
  object types.
```

Reply:
I put "only" there to emphasize how this is different from the `BaseObject` that's listed onthe next line.